src/metabase/lib/drill_thru/summarize_column_by_time.cljc

```diff
@@ -21,20 +24,22 @@
              (not (lib.types.isa/structured? column))
              (lib.types.isa/summable? column))
     ;; There must be a date dimension available.
-    (when-let [breakout-column (->> (lib.breakout/breakoutable-columns query stage-number)
-                                    (filter lib.types.isa/temporal?)
-                                    first)]
-      {:lib/type :metabase.lib.drill-thru/drill-thru
-       :type     :drill-thru/summarize-column-by-time
-       :column   column
-       :breakout breakout-column})))
+    (when-let [breakout-column (m/find-first lib.types.isa/temporal?
+                                             (lib.breakout/breakoutable-columns query stage-number))]
+      (when-let [bucketing-unit (m/find-first :default
+                                              (lib.temporal-bucket/available-temporal-buckets query stage-number breakout-column))]
+        ;; only suggest this drill thru if the breakout it would apply does not already exist.
+        (let [bucketed (lib.temporal-bucket/with-temporal-bucket breakout-column bucketing-unit)]
```

Comment:
Fixed bug: this drill thru was suggested even if the query already had the same exact breakout.